cmd/snap/main_test.go

```diff
@@ -56,12 +58,17 @@ func (s *BaseSnapSuite) SetUpTest(c *C) {
 	snap.Stdin = s.stdin
 	snap.Stdout = s.stdout
 	snap.Stderr = s.stderr
+	s.AuthFile = filepath.Join(c.MkDir(), "json")
+	os.Setenv(TestAuthFileEnvKey, s.AuthFile)
 }
 
 func (s *BaseSnapSuite) TearDownTest(c *C) {
 	snap.Stdin = os.Stdin
 	snap.Stdout = os.Stdout
 	snap.Stderr = os.Stderr
+	c.Assert(s.AuthFile == "", Equals, false)
```

Comment:
fwiw you could also have written this as `c.Assert(s.AuthFile, Not(Equals), "")`, or even `c.Assert(s.AuthFile, Not(HasLen), 0)`